index.html

```diff
@@ -0,0 +1,77 @@
+                <li class="gradlink-style">iPod</li>
+                <li class=" gradlink-style">iPhone</li>
+                <li class="gradlink-style">iPad</li>
+                <li class=" gradlink-style">iTunes</li>
+                <li class="gradlink-style">Support</li> 
+                <li class="gradlinkwithouthover-style">
+                    <i class="fa fa-search ilinksearch" aria-hidden="true"></i><input class="searchbtn" type="search">
+                </li>                   
+            </ul>                
+            <p class="verse-style">What will your verse be?</p>
+            <p class="verseDetails-style">Everyone has something to share. See how people are using </p>
+            <p class="verseDetails1-style">iPad in their unique ways to add to the world’s story.</p>     
+            <p class="verselink-style"><a href="#">Watch the “Your Verse” film and explore the stories 
+                <i class="far fa-play-circle"></i></a>
+            </p>            
+        </div>
+        <div class="imageContainer">
+            <img class="image-styling" src="img/1_iphone5s.jpg" alt="pic1" />
+            <img class="image-styling" src="img/2_iphone5c.jpg" alt="pic2" />
+            <img class="image-styling" src="img/3_30_years.jpg" alt="pic3" />
+            <img class="image-styling" src="img/4_supplier_responsibility.jpg" alt="pic4"/>
+        </div>
+        <div class="Bottomofpage">
+            <div class="footer-style">
+                <ul class="first-footer">
+                    <li class="firstfooterchild1">
+                        <p>Shop the <a href="#">Apple Online Store</a> (1-800-MY-APPLE), visit an <a href="#">Apple Retail</a>
+                            Store, or find a
+                            <a href="#">reseller</a></p>
+                    </li>
+    
+                    <li class="firstfooterchild2">
+                        <a class="border" href="#">sitemap</a>
+                        <a class="border" href="#">Hot-news</a>
+                        <a class="border" href="#">RSS Feeds</a>
+                        <a class="border" href="#">Media Info</a>
+                        <a class="border" href="#">enviroment</a>
+                        <a class="border" href="#">Job Opportunities</a>
+                        <a class="border" href="#">Contact us</a>
+                        <a href="#"><img class="usa-logo" src="img/usa_logo.png" alt="USA logo"></a>
+                    </li>
+                </ul>
+    
+                <ul class="second-footer">
+                    <li>
+                        <p class="second-footer-p">Copyright © 2014 Apple Inc. All rights reserved. <a class="border1"
+                                href="#">Terms of Use</a> <a href="#">Privacy Policy</a>
+                        </p>
+                    </li>
+                </ul>
+            </div>
+        </div>
+    </main>
+    
+    <footer>
+        
+    </footer>
```

Comment:
I see you have footer tags but they are not wrapping the footer section. Kindly add the footer section inside the footer tags

Reply:
Footer in now in the <footer/> section